pages/desktop/discovery.py

```diff
@@ -35,23 +35,9 @@ class DiscoveryPane(Base):
 
     _featured_addons_base_locator = (By.CSS_SELECTOR, '#featured-addons .addon-title ')
 
-    def __init__(self, testsetup, path):
```

Comment:
So have you just done away with this whole `services_base_url` thing? I cannot claim to have ever understood what it was about, but I do see that it still exists in `conftest.py`.